roxie/ccd/ccdserver.cpp

```diff
@@ -386,17 +386,15 @@ class CRoxieServerActivityFactoryBase : public CActivityFactory, implements IRox
     bool optUnordered = false; // is the output specified as unordered?
     unsigned heapFlags;
 
-    mutable __int64 processed;
-    mutable __int64 started;
+    mutable std::atomic<__int64> processed = {0};
```

Comment:
Could use RelaxedAtomic (in jatomic) to simplify some of the code.  Not sure if it is better or not.